src/lib/y2_alternatives/alternative.rb

```diff
@@ -129,7 +129,12 @@ def automatic_mode!
 
     def save
       return unless @modified
-      STATUS_COMMANDS[@status].execute(self)
+      begin
+        STATUS_COMMANDS[@status].execute(self)
+        true
+      rescue Cheetah::ExecutionFailed
```

Comment:
I think this cannot happen as Execute module itself already catch exceptions. of course unless you use cheetah directly.